app/uk/gov/hmrc/helptosavefrontend/views/email/check_your_email.scala.html

```diff
@@ -26,18 +26,22 @@
     bodyClasses = None) {
 
     <h1 class="heading-large">@messages("hts.email-verification.check-your-email.title.h1")</h1>
-    <p class="lede">@messages("hts.email-verification.check-your-email.we-have-sent-mail") <br />@email</p>
+    <p class="lede">@Html(messages("hts.email-verification.check-your-email.we-have-sent-mail")) <br />@email</p>
     <h2 class="heading-medium">@messages("hts.email-verification.check-your-email.what-next")</h2>
-    <p class="content">@messages("hts.email-verification.check-your-email.content1")</p>
+    <p class="content">@Html(messages("hts.email-verification.check-your-email.content1"))</p>
     <p class="content">@messages("hts.email-verification.check-your-email.content2")</p>
-    <p class="content">@messages("hts.email-verification.check-your-email.content3")</p>
+    <p class="content">@Html(Messages("hts.email-verification.check-your-email.content3"))</p>
     <h2 class="heading-medium">@messages("hts.email-verification.check-your-email.where-used-heading")</h2>
-    <p class="content">@messages("hts.email-verification.check-your-email.where-used")</p>
+    <p class="content">@Html(Messages("hts.email-verification.check-your-email.where-used"))</p>
 
```

Comment:
The above looks a bit inconsistent? Is ```@Html(Messages...``` meant to replace all ```messages``` under content and lede occurrences?